### ISSUE TITLE
Error during building - air_sensor.cpp

### ISSUE THREAD

**Issue by ghost** [Zubax/zubax_gnss#13]
Getting following error on the latest commit.

cc1plus: error: -Werror=implicit-fallthrough: no option -Wimplicit-fallthrough
zubax_chibios///chibios/os/common/ports/ARMCMx/compilers/GCC/rules.mk:198: recipe for target 'build/obj/air_sensor.o' failed
make: *** [build/obj/air_sensor.o] Error 1

GNU Tools for ARM Embedded Processors 6.3.1, but tried with 7 version as well with no success.

**Comment by pavel-kirienko** [comment]
> tried with 7 version as well with no success.

Can't reproduce, works for me. Maybe you were using an older version and didn't notice that?

**Comment by ghost** [comment]
Using the 7.2.1 says "This compiler is not supported".

GCC version is below

![gcc_version](https://user-images.githubusercontent.com/37074852/51836058-3f4d7300-2310-11e9-8c7e-ac5efc3aecd8.png)

Build fail error

![build_fail](https://user-images.githubusercontent.com/37074852/51836071-4bd1cb80-2310-11e9-830b-9923e9d190f7.png)

**Comment by pavel-kirienko** [comment]
Are you sure you're building the latest master? 7.2.1 is supported there:

https://github.com/Zubax/zubax_gnss/blob/54f3f8819c087ceb9ff1b4ba8c13d5d9ae2f5028/firmware/src/main.cpp#L224-L226

**Comment by ghost** [comment]
I saw it there. That's why chose the 7.2.1.

yes I am using the latest master as can be seen here.
![git_log](https://user-images.githubusercontent.com/37074852/51837421-5f7f3100-2314-11e9-965b-a1480ccb9018.png)


Any ideas what might be the issue? I was able to build this version [https://github.com/Zubax/zubax_gnss/commit/ae4618de637aaf76766130fc1c723b76af233652] with 6.3.1 though.

**Comment by pavel-kirienko** [comment]
> Using the 7.2.1 says "This compiler is not supported".

This apparently means that you are not really using 7.2.1. Please make sure that your make recipes invoke the correct version of the compiler.

**Comment by bys1123** [comment]
I got this error too, and I tried gcc 7.2.1
I'm sure I'm using this version, I can build PX4, also when I run `arm-none-eabi-gcc --version` I have:

>arm-none-eabi-gcc (GNU Tools for Arm Embedded Processors 7-2017-q4-major) 7.2.1 20170904 (release) [ARM/embedded-7-branch revision 255204]
Copyright (C) 2017 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Don't know why it's not work~
